build/configs/rtl8721csm/rtl8721csm_make_bin.sh

```diff
@@ -171,8 +171,7 @@ function concatenate_binary_with_signing()
 }
 copy_bootloader;
 if [ "${CONFIG_BINARY_SIGNING}" == "y" ];then
-	# Not support binary signing on public repo. So call making binary without binary signing.
-	concatenate_binary_without_signing;
+	concatenate_binary_with_signing;
```

Comment:
Let's split the commit for this.

Reply:
This is not related with other change in the commit. Please split this line from this commit.